src/utils/injectControls.js

```diff
@@ -318,11 +322,16 @@ function createContextMenu() {
     }
 }
 
-function createTopMiddleContent() {
+function getTrustedHTML(htmlText) {
```

Comment:
Can I ask what the reason for including this function when this is all first party html which is hard coded in?

Reply:
Because it was giving an error in the console for not secure content and failing.